app/uploaders/attachment_uploader.rb

```diff
@@ -0,0 +1,11 @@
+class AttachmentUploader < CarrierWave::Uploader::Base
+  storage :file
+  
+  def store_dir
+     "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
```

Comment:
I'm worried that this storage methodology will not work in production as I believe our storage on the host is ephemeral. As such, any stored documents would be deleted any time the host was restarted (e.g. during a deployment)

We'll most likely need to use S3 in order to store anything permanently.

Reply:
Updated: I just checked, and `paperclip` is already configured to use S3 storage. You can find examples in the `members` model file.